File: Knowledge_Demo/Progress_Demo/CustomCell.m
//
//  CustomCell.m
//  Progress_Demo
//
//  Created by 戴运鹏 on 2019/3/2.
//  Copyright © 2019 戴运鹏. All rights reserved.
//

#import "CustomCell.h"
#import "Custom.h"

@interface CustomCell ()
@property (nonatomic,weak)Custom *custom;
@property (nonatomic,weak)Custom *custom1;
@end

@implementation CustomCell

- (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
{
    if (self = [super initWithStyle:style reuseIdentifier:reuseIdentifier]) {
        [self setUI];
    }
    return self;
}

- (void)setUI
{
    Custom *custom = [[Custom alloc]init];
    self.custom = custom;
    [self.contentView addSubview:custom];
    
    Custom *custom1 = [[Custom alloc]init];
    self.custom1 = custom1;
    [self.contentView addSubview:custom1];
  
}

- (void)layoutSubviews
{
    [super layoutSubviews];
    self.custom.frame = CGRectMake(0, 0, [UIScreen mainScreen].bounds.size.width / 2, [UIScreen mainScreen].bounds.size.width / 2);
       self.custom1.frame = CGRectMake([UIScreen mainScreen].bounds.size.width / 2, 0, [UIScreen mainScreen].bounds.size.width / 2, [UIScreen mainScreen].bounds.size.width / 2);
}

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}



@end

File: SwiftExercise/SwiftExercise/CustomCell.swift
//
//  CustomCell.swift
//  SwiftExercise
//
//  Created by 戴运鹏 on 2018/12/2.
//  Copyright © 2018 戴运鹏. All rights reserved.
//

import UIKit

class CustomCell: UITableViewCell {
    var pic: UIImageView?
    var title:UILabel?
    
    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
    super.init(style: style, reuseIdentifier: reuseIdentifier)
        pic = UIImageView.init(frame: CGRect.init(x: 10, y: 10, width: 30, height: 30))
        pic?.backgroundColor = UIColor.green
        self.contentView.addSubview(pic!)
        
        title = UILabel.init(frame: CGRect.init(x: 10, y: 50, width: 100, height: 30))
        self.contentView.addSubview(title!)
//        title?.text = "你好"
        title?.textColor = UIColor.red
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    //
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
